CMakeLists.txt

```
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.18.2 FATAL_ERROR)
```

```suggestion
cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
```
The latest version has required fixes for Arm Compiler 6 support.